cardinal/world.go

```diff
@@ -295,27 +310,31 @@ func (w *World) StartGame() error {
 		return errors.New("game has already been started")
 	}
 
-	if err := w.instance.LoadGameState(); err != nil {
-		if errors.Is(err, ecs.ErrEntitiesCreatedBeforeLoadingGameState) {
+	if err := w.loadGameState(); err != nil {
+		if errors.Is(err, ErrEntitiesCreatedBeforeLoadGameState) {
 			return eris.Wrap(ErrEntitiesCreatedBeforeStartGame, "")
 		}
 		return err
 	}
```

Comment:
actually, i think we can remove ErrEntitiesCreatedBeforeLoadGameState. there are guards on the entity creation functions now